dashboard.php

```diff
@@ -184,18 +184,18 @@
             $('.upload1').show();
```

Comment:
@shank-code why naming like upload1 etc, this's not acceptable variable naming convention